bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/Units.java

```diff
@@ -91,13 +92,15 @@ public final class Units extends CustomUnits {
     public static final Unit<AmountOfSubstance> MOLE = addUnit(tech.units.indriya.unit.Units.MOLE);
     public static final Unit<Volume> LITRE = addUnit(tech.units.indriya.unit.Units.LITRE);
     @SuppressWarnings("unchecked")
-    public static final Unit<AmountOfSubstance> DEUTSCHE_HAERTE = addUnit(new TransformedUnit<>("°dH",
-            (Unit<AmountOfSubstance>) MetricPrefix.MILLI(Units.MOLE).divide(Units.LITRE), MultiplyConverter.of(5.6)));
+    public static final Unit<Dimensionless> DEUTSCHE_HAERTE = addUnit((Unit<Dimensionless>) new TransformedUnit<>("°dH",
```

Comment:
Why is this changed to Dimensionless?

Reply:
See this [comment](https://github.com/openhab/openhab-core/issues/4307#issuecomment-2226858965)